velox/expression/tests/StringWriterTest.cpp

```diff
@@ -103,4 +105,26 @@ TEST_F(StringWriterTest, copyFromCString) {
 
   ASSERT_EQ(vector->valueAt(0), "1 2 3 4 5 "_sv);
 }
+
+TEST_F(StringWriterTest, vectorWriter) {
+  auto vector = makeFlatVector<StringView>(3);
+  exec::VectorWriter<Varchar> writer;
+  writer.init(*vector);
+  writer.setOffset(0);
+  writer.current().copy_from("1 2 3");
+  writer.commitNull();
+
+  writer.setOffset(1);
+  writer.current().copy_from("4 5 6");
+  writer.commit(true);
+
+  writer.setOffset(2);
+  writer.current().copy_from("7 8 9");
+  writer.commit(false);
+  writer.finish();
+
+  auto expected = std::vector<std::optional<StringView>>{
```

Comment:
nit: you can use std::string instead of StringView; this makes code a bit easier to read